Readme.md

```diff
@@ -56,6 +56,27 @@ The built-in converters handle the following O1 content parts:
 
 Additionally, if a custom converter fills in the `OrchardIds` content part's `Parent` property on the generated content item, then it also adds it to the parent content item's `ListPart`.
 
+### Content Sets
+
+Adds an attachable (named) content part that ties together a content item and a set of variants for it. This is similar to Content Localization part, but generic and not tied to the culture options. The `IContentSetEventHandler.GetSupportedOptionsAsync()` extension point is used to generate the valid options for a content item with an attached `ContentSetPart`.
+
+The content items are indexed into the `ContentSetIndex`. The `IContentSetManager` has methods to retrieve the existing content items (or just the index rows) for a specific content set.
+
+When the content part is attached, you will see a new dropdown in the content items list on the admin dashboard, just like it is with the Content Localization part, but this is uses the named part's display text as its label. You can use the dropdown or the editor view to select an options. Then the content item is cloned and that option's key assigned to it.
```

Comment:
Broken tense and `but this is uses the named part's display text as its label.`

Reply:
`You can use the dropdown or the editor view to select an options.` using plural after `an`.